lgc/interface/lgc/CommonDefs.h

```diff
@@ -89,6 +89,12 @@ enum class ResourceNodeType : unsigned {
   DescriptorReserved16,
   Count, ///< Count of resource mapping node types.
 };
+
+// Represents mapping layout of the resources used in shaders
+enum class ResourceMappingLayout : unsigned {
+  Compact = 0, /// Compact scheme make full use of all the user data registers.
+  Indirect     /// The searching path of resource belongs to a specific binding is fixed in indirect scheme.
```

Comment:
The same enumeration was added in driver, which changes the resource mapping layout. I think it's best to keep their names and comments consistent. Also this enumeration is not only for push Constant, the whole resource mapping is affected. Thanks.

Reply:
At the very least, could up update the comment above, even if you can't update the name.  If I am reading code that checks if the layout is indirect, I have no way of knowing what that means because the comment references itself: what is search path for the?  The comment for indirect does not read well.  I'm guessing "belongs" is a typo and should be "belonging".

Can we ask the driver team to update the comment in the driver code?  Then you can keep them the same.

Reply:
"The fixed user data registers. When the pipeline layout only contains part of layout information of the final executable pipeline, we don't know the offset of user data register, so we must use fixed offset." 

Is it feasible to change the comment like above?